doc/vault-integration.md

```diff
@@ -222,7 +226,10 @@ In the [claim file](../examples/aws-provider/composite-resources/s3/multi-tenant
 Since we created a provider config named `application1-provider-config`, we should be able to create a claim in namespace called application1. 
 
 ```bash
-kubectl apply -n application1 examples/aws-provider/composite-resources/s3/multi-tenant.yaml
+#create namespace
+kubectl create ns application1
```

Comment:
this could fail if the ns exists. a fail safe alternative could be the following 

```
kubectl create namespace application1 --dry-run=client -o yaml | kubectl apply -f -
```

Reply:
I can append a `|| true`